platform/switch/detect.py

```diff
@@ -11,16 +11,14 @@ def get_name():
 
 def can_build():
     # Check the minimal dependencies
-    if not os.path.exists('/opt/devkitpro/devkitA64'):
-        print("devkitA64 not found.. switch disabled.")
+    if "DEVKITPRO" not in os.environ:
         return False
 
-    err = os.system("pkg-config --version > /dev/null")
```

Comment:
if host pkg-config is not installed build will still fail - re-add this check

Reply:
True, I didn't notice the DEVKITPRO one was a wrapper. I guess we can also define PKG_CONFIG_PATH and use the system one. But don't care either way.